test/unit/storage/mapping/FixedContentTypeMapper.test.ts

```diff
@@ -183,6 +183,15 @@ describe('An FixedContentTypeMapper', (): void => {
         await expect(mapper.mapFilePathToUrl(`${rootFilepath}test`, false)).rejects.toThrow(NotFoundHttpError);
         await expect(mapper.mapFilePathToUrl(`${rootFilepath}test.txt`, false)).rejects.toThrow(NotFoundHttpError);
       });
+
+      it('returns a generate file path for metadata regardless of the suffix.', async(): Promise<void> => {
```

Comment:
```suggestion
      it('returns a generated file path for metadata regardless of the suffix.', async(): Promise<void> => {
```

Reply:
Thank you! I totally did not notice this. There is a PR to fix it. I wish I had noticed it myself yesterday. :sweat: